studio/components/interfaces/Settings/Addons/ComputeInstanceSidePanel.tsx

```diff
@@ -430,6 +441,20 @@ const ComputeInstanceSidePanel = () => {
                 <p>You need to disable PITR before downgrading to Micro Compute.</p>
               </Alert>
             )}
+
+            {hasChanges &&
+              subscription?.billing_via_partner &&
+              subscription.scheduled_plan_change?.target_plan !== undefined && (
+                <Alert_Shadcn_ variant={'warning'} className="mb-2">
+                  <IconAlertTriangle className="h-4 w-4" />
+                  <AlertDescription_Shadcn_ className="flex justify-between items-center">
```

Comment:
these classes might not do anything

Reply:
They do - button is on the right and spaced out. Wanted to have text + cta on the right, if we have another way of doing it with the alert from shadcn happy to adjust